File: android/app/src/main/java/com/gwangy/nassspandroidsample/NapSspInitializer.kt
package com.gwangy.nassspandroidsample

object NapSspInitializer {
    fun initialize() {
        val context = AppContextHolder.appContext ?: return
        NapSspSdkIntegration.initialize(context)
        val adUnitIds = NapSspConfig.adUnitIds(context)
        println("NapSsp Android init mediaKey=${NapSspConfig.mediaKey(context)} adUnitIds=${adUnitIds.values.joinToString(",")}")
        println("NapSsp Android mediation hints=${NapSspConfig.MEDITATION_HINTS.joinToString { "${it.first}:${it.second}" }}")
    }
}

File: ios/Sources/NapSspIOSSample/NapSspInitializer.swift
import Foundation

enum NapSspInitializer {
    static func initialize() {
        NapSspSdkIntegration.initializeSdk()
        print("NapSsp iOS init mediaKey=\(NapSspConfig.mediaKey) adUnitIds=\(NapSspConfig.adUnitIDs.values.joined(separator: ","))")
        print("NapSsp iOS mediation hints=\(NapSspConfig.mediationHints.map { $0.0 + ":" + $0.1 }.joined(separator: ","))")
    }
}
